examples/extended_validation/src/tests.rs

```diff
@@ -0,0 +1,89 @@
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::{ContentType, Status};
+
+use super::rocket;
+
+fn test_login(username: &str,
```

Comment:
Well, we have a `rustfmt` file that kinda does what I want it to do, but otherwise, no. `rustfmt` still isn't super dependable, unfortunately.